### PR TITLE
Add test case in CandidateTotalAggregateView

### LINKED ISSUE #5092: Add test case in /candidates/totals/aggregates/
### What we’re after
Add test case in the endpoint: /candidates/totals/aggregates/

### Related ticket(s)
[https://github.com/fecgov/openFEC/pull/5090](https://github.com/fecgov/openFEC/pull/5090)

### Completion criteria
- [ ] Add test case in endpoint
- [ ] pass pytest

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)
After create new endpoint: /`candidates/totals/aggregates/`, we need add test cases.

- Resolves #5092
- Ref PR #5090 

### Required reviewers
1 developer

## Impacted areas of the application
endpoint: /`candidates/totals/aggregates/`

### Completion criteria
- [x] Add test case in endpoint
- [x] pass pytest
- [x] Add one more filter: district
- [x] Fix deprecated function from query.add_colum() to query.add_colums()
- [x] Change endpoint name from `/candidates/totals/aggregate/` to `/candidates/totals/aggregates/`

## How to test
- Checkout branch, point to dev db
- `pytest`
- test endpoint /`candidates/totals/aggregates/` by passing each aggregate_by parameter.
Screen Shot 2022-04-07 at 5 41 00 PM

### CODE REVIEW

**Comment by cnlucas** [comment]
@fec-jli, pytest passed for me. I'm having some issues accessing the data. A few small items that might be helpful, but are not really about the test. 
1. Required fields perhaps should be marked with a *required 
2. Integer is being shown as $int32 for the first several filters

**Comment by fec-jli** [comment]
> @fec-jli, pytest passed for me. I'm having some issues accessing the data. A few small items that might be helpful, but are not really about the test.
> 
> 1. Required fields perhaps should be marked with a *required
> 2. Integer is being shown as $int32 for the first several filters
> 3. Perhaps making the election_full description a bit more clear
> 
> The test looks great and the updates to the names look good. I'm so impressed with your hard work and skill.

@cnlucas Thank you for your review and comments. 
1. There is no required field for this endpoint.
2. When we define the field type = fields.Int, it will show $int32 which means 32 bits integer type, not big integer. this affect all endpoints.
3. I agree the election_full description is obscure. Could you create a ticket to address this? All endpoints share the election_full description now.
Thanks.

**Review by cnlucas — APPROVED** [review]
Thanks @fec-jli, will create the ticket for the election_full.

File: webservices/args.py [hunk 1]
```diff
@@ -1017,14 +1017,16 @@ def make_seek_args(field=fields.Int, description=None):
 candidate_total_aggregate = {
     'election_year': fields.List(fields.Int, description=docs.RECORD_CYCLE),
     'office': fields.Str(validate=validate.OneOf(['', 'H', 'S', 'P']), description=docs.OFFICE),
+    'party': fields.Str(validate=validate.OneOf(['DEM', 'REP', 'Other']), description=docs.PARTY),
```
Comment (hcaofec) [inline webservices/args.py:1020]: Should the party filter be removed since the logic is not in resource yet?

**Review by hcaofec — APPROVED** [review]
Great work on this endpoint!

**Pull request merged**